src/views/Room.vue

```diff
@@ -108,7 +109,7 @@ export default {
 
       rtc.on("local_stream_error", (error) => {
         logger.error("local stream error", error)
-        this.waiting = false
+        this.screenMessage = null
         this.uiState = [UserMediaConfigurator, { error: "local_stream_error" }]
```

Comment:
screenmessage = null and uistate change always go together, right? I guess it would be a lot easier then to make this into a function, even though it always only merges 2 lines into 1?